### DISCUSSION TITLE
Test Runner Chrome 119 Issues

### DISCUSSION THREAD

**Discussion by rfinken** [cypress-io/cypress#28221] [Questions and Help]
All of a sudden the cypress test runner will not run my tests when I choose Chrome 119. Edge and Electron work fine. When I run the test I get a loading spinner and then the test runner page flashes over and over and over. I finally have to kill the connection to get it to stop. Anyone else have this issue?

**Comment by Mehran-Badami** [comment]
Hello,
I am experiencing the same issue. My system information is as follows:
Chrome version: 119
Node version: v20.9.0
Cypress package version: 13.6.0
Cypress binary version: 13.6.0
Electron version: 25.8.4
Bundled Node version: 18.15.0

**Comment by ghost** [comment]


**Reply by Mehran-Badami** [reply]
Hello,
I have done all the tasks you mentioned, but unfortunately, I still have the problem.
I have even downgraded Cypress to your version.

**Comment by thomastrapanese** [comment]
I have the same issue with cypress 13.3.3 and chrome 119

**Comment by jennifer-shehane** [comment]
Are you experiencing this issue? https://github.com/cypress-io/cypress/issues/28285

![http://g.recordit.co/T4YoJyg6rc.gif](http://g.recordit.co/T4YoJyg6rc.gif)

**Reply by ghost** [reply]
that's not what I was seeing, in my case it would only stop. Different issue then, deleting my previous comment.

**Reply by Mehran-Badami** [reply]
I am experiencing the same issue.
